src/components/fieldHelpers/withHorizontalLabel.tsx

```diff
@@ -32,7 +36,15 @@ const WithHorizontalLabel: FC<Props> = ({
     >
       {children}
     </span>
-    <span className="w-12/12">{renderContent && renderContent()}</span>
+    <label
+      htmlFor={name}
+      className={classNames("w-12/12", {
+        "cursor-not-allowed text-grey-4": disabled,
+        "cursor-pointer": !disabled,
+      })}
+    >
+      {renderContent && renderContent()}
+    </label>
```

Comment:
do we really want to render an empty `<label></label>` when `renderContent` is not passed?

Reply:
alternatively, make `renderContent` a required prop (we might use it like this anyway already) and drop the conditional